### ISSUE TITLE
Ability to exclude schema attribute from resource or data source in generator config

### ISSUE THREAD

**Issue by pauly4it** [hashicorp/terraform-plugin-codegen-openapi#81]
Similar to how you can override an attribute description in the generator config, it would be useful to be able to exclude an attribute or sub-attribute from a resource or data source.

**Use cases**
* Exclude an attribute related to functionality we do not want to expose via the Terraform provider
* Exclude an attribute that uses a schema type not yet supported by the codegen tool or which is causing generation errors

**Example**
A System resource post request has the following:
```yaml
SystemPostRequest:
  properties:
    feature_a:
      $ref: '#/components/schemas/SystemFeatureAConfig'
    feature_b:
      description: Feature B is enabled
      type: boolean
    feature_c:
      $ref: '#/components/schemas/SystemFeatureCConfig'    
    name:
      description: system name
      type: string
  required:
    - name
SystemFeatureAConfig:
  properties:
    customization:
      $ref: '#/components/schemas/SystemFeatureACustomization'
    enforced:
      description: true if the feature is enforced
      type: boolean
    type:
      description: enforcement type
      type: string
  required:
    - enforced
    - type
SystemFeatureACustomization: {}
SystemFeatureBConfig:
  properties:
    enforced:
      description: true if the feature is enforced
      type: boolean
    type:
      description: enforcement type
      type: string
  required:
    - enforced
    - type
SystemFeatureCConfig:
  # .. schema
```

For this resource, let's say we don't want to expose the configuration of Feature C via the Terraform provider, and we want to disable Feature A's `customization` because it's an Any Type schema not currently supported by the codegen tool.

**Comment by austinvalle** [comment]
This request seems like a good idea! There are definitely limitations to what can be generated (especially early on in the tech preview), so providing this escape hatch can remove a lot of headache 👍🏻 

I'm thinking maybe a similar "flattened" style to the overrides, like:
```yaml
data_sources:
  gists:
    read:
      path: /gists/{gist_id}
      method: GET
    schema:
      ignores:
        # Ignore a specific property in an object
        - fork_of.forks
        # Ignore an entire object and all child properties
        - fork_of
        # Ignore a specific property at the root
        - comments_url
```

I think it's possible that in the future, it might be desired to ignore specific properties just on the request/response body for create/read/update/delete, so maybe will need to consider how best to introduce this configuration 🤔

**Comment by austinvalle** [comment]
This has been released with [`v0.2.0`](https://github.com/hashicorp/terraform-plugin-codegen-openapi/releases/tag/v0.2.0)
```bash
 $ go install github.com/hashicorp/terraform-plugin-codegen-openapi/cmd/tfplugingen-openapi@latest                                  
go: downloading github.com/hashicorp/terraform-plugin-codegen-openapi v0.2.0
```

Documentation for this functionality can be found on the HashiCorp developer site here: https://developer.hashicorp.com/terraform/plugin/code-generation/openapi-generator#ignoring-oas-properties
